torch/csrc/jit/python/init.cpp

```diff
@@ -1030,6 +1049,14 @@ void initJITBindings(PyObject* module) {
         }
       });
 
+  py::class_<c10::SymbolicIntNode, std::shared_ptr<c10::SymbolicIntNode>>(m, "SymbolicIntNode")
```

Comment:
@ezyang This is hopefully a smaller issue but since we are binding `SymbolicIntNode` , there's no easy way to extract the wrapper python object (`obj`) from a bound `SymbolicIntNode`. We could probably add a method to do it, but it would need to down cast to `PythonSymbolicIntNode`. @Chillee told me that they may not even need the wrapped python object as long as `SymbolicIntNode` redispatches all the ops that we want to trace.

Reply:
Yeah, this is one of the things in the UX that is simpler if the inner and the outer python objects are the same; you can use the normal `isinstance` test in Python to retrieve the internals. In properly written code this should never be necessary but it probably is useful for debugging purposes. You should probably add the unsafe downcast and also a way to test if the symint is a python symint or not.